### ISSUE TITLE
[Bug]: 如果一个笔记/文件夹重命名，同步会生成两份数据

### ISSUE THREAD

**Issue by seyeeL** [remotely-save/remotely-save#608]
### What happened?

这个问题能够解决吗，不然还挺难受的

### What OS are you using?

Windows, macOS, iOS (iPhone, iPad)

### What remote cloud services are you using? (Please choose the specified one if it's in the list)

S3 (阿里云 OSS Alibaba Cloud OSS)

### Version of the plugin

0.4.16

### Version of Obsidian

1.5.8

### Using password or not

- [ ] Yes.

### Ensure no sensitive information

- [X] I ensure that no sensitive information is submitted in the issue.

**Comment by seyeeL** [comment]
> 不太符合预期，新建测试 vault 能复现吗？导出同步计划看看咋回事吧

没新建vault，但是出现很多次了。可能是因为我多端同步，并且选择了双向同步的原因？比如我在A设备重命名了，但是B设备还有重命名前的文件，然后隔一段时间B设备触发同步，A也触发，就变成了两份。

**Comment by fyears** [comment]
按道理也是应该可以正确处理的。

> 然后隔一段时间B设备触发同步，A也触发，就变成了两份。

假如 b 对旧文件夹没有改动，那么 b 同步不会影响那个文件夹。从而，a 同步应该可以正确删除云端旧文件夹。如果这里不是如我所述运作，那么确实有问题。

假如 b 对旧文件夹有改动，那么这里确实是无法避免，因为你改动了 b 的旧文件夹，也创建了 a 的新文件夹，这种复杂情况，插件是判断不了的。


建议还是给一下正确复现路径哈。可以多端都导出同步计划。

此外，记得所有端都更新到最新版本，

**Comment by fyears** [comment]
dup of #611

**Comment by nallebrean** [comment]
Well I have used Google translate to follow this issue. I have similar problem. I have multiple units. I moved at lot of files from directory A to a new B. Works fine on unit x and y but z did not understand that some of the files were moved. Those files are back on all units now.

Last time when I had to rearrange 2000 files I needed to take some at the time and let it sync. Otherwise files started to be doubled.

However I am surprised that it works this time with approx 200 files on 2 on unit x and y but not when z are syncing.

Z had a 4 day old sync and the other 2 where up to date.
